App.js

```diff
@@ -1,114 +1,45 @@
-/**
- * Sample React Native App
- * https://github.com/facebook/react-native
- *
- * @format
- * @flow strict-local
- */
-
+import 'react-native-gesture-handler';
 import React from 'react';
-import {
-  SafeAreaView,
-  StyleSheet,
-  ScrollView,
-  View,
-  Text,
-  StatusBar,
-} from 'react-native';
+import {NavigationContainer, useLinkProps} from '@react-navigation/native';
+import {createStackNavigator} from '@react-navigation/stack';
+import {appCalculadora} from './components/calculator/index';
+import {historialCalculadora} from './components/history/index';
 
-import {
-  Header,
-  LearnMoreLinks,
-  Colors,
-  DebugInstructions,
-  ReloadInstructions,
-} from 'react-native/Libraries/NewAppScreen';
+const STACK = createStackNavigator();
 
-const App: () => React$Node = () => {
+function appTrainingMobile() {
```

Comment:
```suggestion
const  AppTrainingMobile = () =>  {
```
Por qué lo cambiaste?